uploads/sample_labels_to_annotate.csv

```diff
@@ -1,12 +1,29 @@
-id,name,label,description,type,values,parent,annotations,tags
```

Comment:
We are not only receiving the 'labels' but a csv file that could contain field 'type', 'description', etc... 
So the original file contains the possible CSV format. You can add more labels to the same file format.

Reply:
do you suggest me to add more 'columns' there?